engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-host-bulk-module-streams-modal.html

```diff
@@ -9,7 +9,6 @@ <h3 translate ng-hide="selectedModuleStreamOption === 'available'">Available Mod
       <input type="hidden" name="remote_action" ng-value="moduleStreamActionFormValues.remoteAction"/>
       <input type="hidden" name="module_stream_action" ng-value="moduleStreamActionFormValues.moduleStreamAction"/>
       <input type="hidden" name="module_spec" ng-value="moduleStreamActionFormValues.moduleSpec"/>
-      <input type="hidden" name="scoped_search" ng-value="moduleStreamActionFormValues.search"/>
```

Comment:
`search` is being set in `content-hosts-bulk-module-streams-modal.controller.js`. Should we put back the above line in content-host-bulk-module-streams-modal.html ?

https://github.com/Katello/katello/blob/4a99b51048fc0816c3b4237f91448b39ae9af388/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-module-streams-modal.controller.js#L42

Reply:
I'll need to look into this but it appears there's some issue with the modularity modal - will need to take a look at that

Reply:
Actually, I think it's dead code so I'll remove it from the controller